tests/utils.py

```diff
@@ -263,10 +256,11 @@ def __init__(self, lightning_dir, bitcoin_dir, port=9735, random_hsm=False):
 
         # Last 32-bytes of final part of dir -> seed.
         seed = (bytes(re.search('([^/]+)/*$', lightning_dir).group(1), encoding='utf-8') + bytes(32))[:32]
+        if not random_hsm:
+            with open(os.path.join(lightning_dir, 'hsm_secret'), 'wb') as f:
```

Comment:
Failing in CI, probably because the `lightning_dir` might not exist yet at this point.

Reply:
Assuming this comment predates the latest version, since the call to `makedirs` is 5 lines above this :wink: